src/nix/install.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+# Move to the same directory as this script
+set -e
+FEATURE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "${FEATURE_DIR}"
+
+# Option defaults
+VERSION="${VERSION:-"latest"}"
+MULTIUSER="${MULTIUSER:-"true"}"
```

Comment:
Can we remove all such variable assignments as well. Thanks!

Reply:
See prev comment